+nosnoc/+activeset/Heaviside.m

```diff
@@ -0,0 +1,44 @@
+classdef Heaviside < handle
+    properties(SetAccess=private)
+        I_0
+        I_1
+        I_free
+        times
+        stages
```

Comment:
stages of what? rk stages, control stages, other stages? please explain. 
if is a chunk of the horizon where the active set is not changed, we may call it phase instead, coming from multi-phase OCP which is a more common name

Reply:
Added a comment. I am happy to change the name to phases.

Reply:
At this point i still dont understand what are stages :D 
If they are phases, esentially paris of active set and time interval, then we should name them phases. if they are some other sort of stages. should be described in more detail.

Reply:
Stages is the name for the alternate approach for defining the end points that you wanted: i.e. instead of times you pass `(control_stage, finite_element)` index pairs.

Reply:
In the pass_pss_active_set example there is an example of use. Please add a comment explaining how it is done.